.env.example

```diff
@@ -55,7 +55,8 @@ CLOSE_PAYMENT_RETRY_EVENT_BASE_INTERVAL_SECONDS=60
 CLOSE_PAYMENT_RETRY_EVENT_MAX_ATTEMPTS=3
 NOTIFICATION_RETRY_EVENT_BASE_INTERVAL_SECONDS=600
 NOTIFICATION_RETRY_EVENT_MAX_ATTEMPTS=3
-PERSONAL_DATA_VAULT_API_KEY=personal-data-vault-api-key
+ECOMMERCE_PERSONAL_DATA_VAULT_API_KEY=personal-data-vault-api-key
+SHARED_PERSONAL_DATA_VAULT_API_KEY=personal-data-vault-api-key-wallet-session
```

Comment:
proposal, why not rename everything to `wallet-session`? even if we refer to `shared`, it is  the domain to which the wallet-session-pagopa component belongs

Reply:
I agree, i'm going to rename it

Reply:
or PAGOPA_SESSION :-)

Reply:
pr PAGOPA_PLATFORM_SESSION :-)

Reply:
I correct myself :-)  better `WALLET_SESSION_` which reflects the name of the secret in KV (and we leave it equal to the name defined in the SHARED domain).
what do you think about it? @giovanniberti @CianoDanilo

Reply:
Changed prefix to WALLET_SESSION